README.rst

```diff
@@ -119,7 +112,7 @@ portable version you should observe the following install steps.*
 
        > easy_install openslides-x.x.tar.gz
```

Comment:
Yes, pip already available after Python 3.4 installation on windows. See https://docs.python.org/3/whatsnew/3.4.html#whatsnew-pep-453
So you can replace `easy_install`to `pip`